File: Client/src/main/java/p2pclient/dto/ResChunkDto.java
package p2pclient.dto;

import java.util.List;

public class ResChunkDto {
    public Integer chunk_id;
    public Long chunk_index;
    public String chunk_hash;
    public Integer chunk_size;
    public List<ResPeerDto> chunk_peers;

    public ResChunkDto() {}
}

File: Server/dto/resChunkDto.py
from pydantic import BaseModel
from .resPeerDto import ResPeerDto
from typing import List
from typing import Optional

class ResChunkDto(BaseModel):
    chunk_id : int
    chunk_index : int
    chunk_hash : str
    chunk_size : int
    chunk_peers : List[ResPeerDto] 
